salt/journal/config/srv-journal-local-demo-app-config-parameters.yml

```diff
@@ -40,3 +40,5 @@ parameters:
     subject_rewrites: []
 
     cache_control: private, no-cache, no-store, must-revalidate
+
+    feature_digest_channel: true
```

Comment:
It probably doesn't happen, but there is the potential for critical CSS about the digest channel to spill over into the . CSS should be compatible with both "with digest channel" and "without digest channel" versions anyway if we are feature flagging it.

Reply:
Good news so far is that there's no new CSS.

Reply:
#patternsAtWork #thisIsWhatPatternLibraryLooksLike